app/views/maintaining/contributor_teams/new.html.haml

```diff
@@ -6,17 +6,17 @@
 
     %section.form-section
       = form_with url: maintaining_contributor_teams_path, method: :post, model: @team do
-        = fields_for @team do |fu|
+        = fields_for @team do |f|
```

Comment:
You shouldn't need this. You can elevate the `|f|` to the line above. The fields for is only if you have nested models in one form.